import/src/test/java/com/google/cloud/healthcare/imaging/dicomadapter/CFindServiceTest.java

```diff
@@ -117,6 +120,28 @@ public JSONArray qidoRs(String path) throws DicomWebException {
       }
     }, Status.NotAuthorized);
   }
+  
+  @Test
+  public void testCFindService_withFuzzyMatching() throws Exception {
+	cFINDFlags.fuzzyMatching = true;
+    basicCFindServiceTest(new TestUtils.DicomWebClientTestBase() {
+      @Override
+      public JSONArray qidoRs(String path) throws DicomWebException {
```

Comment:
Can we assert that the path contains fuzzymatching=true? By using something like assertThat(path).isEqualTo("expected value") within the qidoRS function? Similarly for the other test can we assert that the path does not contain fuzzymatching=true?

Reply:
done.